zipkin-courrelate/src/main/scala/org/coursera/zipkin/courrelate/CourrelateProducer.scala

```diff
@@ -0,0 +1,42 @@
+package org.coursera.zipkin.courrelate
+
+import java.util.Properties
+
+import com.twitter.logging.Logger
+import kafka.javaapi.producer.Producer
+import kafka.producer.KeyedMessage
+import kafka.producer.ProducerConfig
+import play.api.libs.json.Json
+
+import scala.collection.JavaConverters._
+
+class CourrelateProducer {
+  val logger = Logger.get(classOf[CourrelateProducer])
+
+  import Formats._
+
+  private[this] val producer = {
+    val downstreamKafkaBrokerList = System.getProperty("downstreamKafka.metadata.broker.list")
```

Comment:
maybe use the `flag` code that the rest of zipkin uses?

https://github.com/twitter/zipkin/blob/29e0f90a8d19a6c80eaf0e7504f72058cd05b7a6/zipkin-receiver-kafka/src/main/scala/com/twitter/zipkin/receiver/kafka/KafkaSpanReceiver.scala#L20